innopoints/views/activity.py

```diff
@@ -37,7 +54,7 @@ def create_activity(project_id):
         db.session.commit()
     except IntegrityError as err:
         db.session.rollback()
-        print(err)  # TODO: replace with proper logging
+        log.error(str(err))
```

Comment:
Use `logging.exception` for logging exceptions as it includes the stack trace as well.
Also, no need to `str` it manually, just pass the object itself.